packages/x-charts/src/LineChart/AreaPlot.tsx

```diff
@@ -47,62 +47,66 @@ const useAggregatedData = () => {
   const defaultYAxisId = yAxisIds[0];
 
   return stackingGroups.flatMap(({ ids: groupIds }) => {
-    return groupIds.flatMap((seriesId) => {
-      const {
-        xAxisKey = defaultXAxisId,
-        yAxisKey = defaultYAxisId,
-        stackedData,
-        data,
-        connectNulls,
-      } = series[seriesId];
-
-      const xScale = getValueToPositionMapper(xAxis[xAxisKey].scale);
-      const yScale = yAxis[yAxisKey].scale;
-      const xData = xAxis[xAxisKey].data;
-
-      const gradientUsed: [string, 'x' | 'y'] | undefined =
-        (yAxis[yAxisKey].colorScale && [yAxisKey, 'y']) ||
-        (xAxis[xAxisKey].colorScale && [xAxisKey, 'x']) ||
-        undefined;
-
-      if (process.env.NODE_ENV !== 'production') {
-        if (xData === undefined) {
-          throw new Error(
-            `MUI X Charts: ${
-              xAxisKey === DEFAULT_X_AXIS_KEY
-                ? 'The first `xAxis`'
-                : `The x-axis with id "${xAxisKey}"`
-            } should have data property to be able to display a line plot.`,
-          );
+    return [...groupIds]
+      .reverse() // Revert staked area for a more pleasant animation
```

Comment:
minnor typo on comment
```suggestion
      .reverse() // Revert stacked area for a more pleasant animation
```